db/migrate/20250121200324_add_sso_id_to_users.core_data_connector.rb

```diff
@@ -0,0 +1,6 @@
+# This migration comes from core_data_connector (originally 20250121194744)
```

Comment:
I might have messed this up. I actually added this migration in my last [PR](https://github.com/performant-software/core-data-cloud/pull/376) in order to prevent any issues with the column not existing and trying to access it in controllers/models. I _did not_, however actually run the migration locally to update `schema.rb`.

I think you can remove this migration, but keep the `schema.rb` change.

Reply:
Looks like my `schema.rb` version wouldn't match the migration that's checked into `develop`.

I think I'll:
* roll back the migration locally
* take my migration out of my branch
* copy the one from `develop` into my branch
* run the migration

Does that sound right?

Reply:
Done.